org.abapgit.adt.backend/src/org/abapgit/adt/backend/internal/FileService.java

```diff
@@ -41,9 +41,9 @@
 public class FileService implements IFileService {
 
 	@Override
-	public String readLocalFileContents(IAbapGitFile file, IExternalRepositoryInfoRequest credentials, String destinationId)
+	public String readLocalFileContents(IAbapGitFile file, String destinationId)
 			throws IOException {
-		return readFileContents(file, LOCAL_VERSION, credentials, destinationId);
+		return readFileContents(file, LOCAL_VERSION, null, destinationId);
```

Comment:
Since reading local file does not require credentials, null value is pass for credentials to the `readFileContents` method call.